docs/conf.py

```diff
@@ -15,7 +15,46 @@
 
 # -- Project information -----------------------------------------------------
 
+import os
+import sys
+import tempfile
+import importlib
+import subprocess
+
+sys.path.append(os.path.abspath('..'))
+
 import honeycomb
+import services
+import integrations
+
+from honeycomb.servicemanager import base_service
+from honeycomb.integrationmanager import integration_utils
+
+services_list = next(os.walk('../services'))[1]
+integrations_list = next(os.walk('../integrations'))[1]
+
+
+def install_plugins(dir, plugins, target):
+    for plugin in plugins:
+        reqs = os.path.join(dir, plugin, "requirements.txt")
+        if os.path.exists(reqs):
+            pipargs = ["install", "-r", reqs, "--target", target, "--ignore-installed"]
+            subprocess.check_call([sys.executable, "-m", "pip"] + pipargs)
+
+
+venv = tempfile.mkdtemp()
```

Comment:
This doesn't look like plugin documentation.

Regardless, wouldn't you prefer creating your venv tempdir under a specific named subdir?

Reply:
nah it's just for sphinx build, doesn't matter